woodwork/tests/accessor/test_statistics.py

```diff
@@ -453,21 +453,21 @@ def test_describe_accessor_method(describe_df):
     # Test numeric columns with nullable ltypes
     numeric_data = describe_df[['numeric_col']]
     for ltype in nullable_numeric_ltypes:
-        expected_vals = pd.Series({
+        expected_vals = pd.Series({  # fix when https://github.com/pandas-dev/pandas/issues/42626 gets resolved
```

Comment:
Is this comment and the one below still relevant?

Reply:
nope!